lib/absinthe/relay/node.ex

```diff
@@ -200,6 +200,9 @@ defmodule Absinthe.Relay.Node do
     end
   end
 
+  @doc """
+  Similar to `from_global_id/2` but raises `RuntimeError` if fails to decode global ID.
```

Comment:
@bruce Not sure if `RuntimeError` is correct in this context (same with `from_global_id!/2`). What do you think?